src/couch/src/couch_server.erl

```diff
@@ -72,12 +73,16 @@ get_uuid() ->
     end.
 
 get_stats() ->
-    {ok, #server{start_time=Time,dbs_open=Open}} =
-            gen_server:call(couch_server, get_server),
+    Fun = fun(N, {TimeAcc, OpenAcc}) ->
+        {ok, #server{start_time=Time,dbs_open=Open}} =
+            gen_server:call(couch_server(N), get_server),
+        {max(Time, TimeAcc), Open + OpenAcc} end,
```

Comment:
"Time" here is a formatted datetime string, so 'max' is a bit arbitrary. all the couch_server_X's are started at the same time, not sure we care?